MongoDB/node.updateOne.js

```diff
@@ -0,0 +1,28 @@
+const _ = require('lodash');
+let logger = global.logger;
+
+module.exports = async (connectorData, inputData) => {
+    logger.trace("MongoDB UpdateOne Node: Invoked!");
+    try {
+        logger.trace('MongoDB updateOne Node: Input - ${JSON.stringify(inputData)}');
+
+        if (_.isEmpty(inputData.filter)) {
+            throw new Error('Filter is empty');
+        }
+        if (_.isEmpty(inputData.update)) {
+            throw new Error('Update data is empty');
+        }
+
+        const result = await connectorData.db.collection(inputData.collection).updateOne(inputData.filter, { $set: inputData.update });
```

Comment:
This update operation assumes that you are updating part of the document. What if i want to replace the whole document? There should be a check for that too.

Read more about replace here - https://www.mongodb.com/docs/drivers/node/current/usage-examples/replaceOne/

But i would prefer findOneAndReplace - https://mongodb.github.io/node-mongodb-native/6.7/classes/Collection.html#findOneAndReplace or